### DISCUSSION TITLE
[PSD10] Esercizio Slide 14

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#170] [Esercizi (A-L)]
Usare mux 2-1 per realizzare le funzioni logiche dell'Esercizio 2.18 (#159 ):

![image](https://user-images.githubusercontent.com/69389461/210059660-2033ee68-825c-4a30-add7-da1b290f0469.png)

**Comment by Elia-Belli** [comment]
Dalle funzioni semplificate si ottiene:
- (a) $Y = C + \bar B$
![a](https://user-images.githubusercontent.com/107944476/211671361-bc8f6386-2eca-4f78-b7eb-a6adfeded2e0.jpg)
- (b) [Soluzione](#issuecomment-1397190550) di @fraxb04 corretta
- (c) $Y = B\bar D E + \bar A B \bar C D+ ABCD$
![MUX2](https://user-images.githubusercontent.com/107944476/211671171-14ed1f24-37fa-452e-8615-e5dc6a3ed811.jpg)

**Comment by fraxb04** [comment]
corretto, tuttavia la seconda equazione è errata, ed è corretta come avevi scrirto nell'esercizio precedente.
ps: considera il gruppo di mux sulla destra
![image](https://user-images.githubusercontent.com/122573871/213487750-468716f5-91b6-47b8-92a8-7a510510a006.png)

**Comment by fraxb04** [comment]
@Elia-Belli 
per il terzo esercizio ho fatto una variante
![image](https://user-images.githubusercontent.com/122573871/213497336-cfae18aa-37d6-4838-b319-bc51ac66f2e3.png)

**Comment by Simone0401** [comment]
Per l'equazione **_c)_** ho provato ad usare meno MUX, tuttavia non so se la mia soluzione è corretta.
<img width="943" alt="immagine" src="https://github.com/user-attachments/assets/d27726fa-99c6-4c8e-b30a-3b7eb433ed02" />
